lib/angular/directive.js

```diff
@@ -32,10 +39,30 @@ angular.module('wfm.file.directives', [])
       scope.$watch('uid', function(uid) {
         initPromise.then(function(cloudUrl) {
           element[0].src = cloudUrl + config.apiPath + '/get/' + uid;
-          console.log(element[0].src);
         });
       });
     }
   };
 })
+  .directive('fileDetail', function($templateCache) {
+    /*
+     detail view of a file, passed in is file object and options.
+     Options help to chose which details we want to render
+     options e.g. {id: true, name: true, uid: true, owner: true, preview:true};
+     */
+    return {
+      restrict: 'E',
+      template: $templateCache.get('wfm-template/file-detail.tpl.html'),
+      scope: {
+        file: '=',
+        displayOptions: '='
+      },
+      controller: function($scope) {
```

Comment:
Can you move the controller to a separate file. This is the general practice we are using for raincatcher modules.